### DISCUSSION TITLE
Multiple calendar

### DISCUSSION THREAD

**Discussion by hariangr** [spatie/laravel-google-calendar#220] [Q&A]
Hey, I have a Google Account that have multiple calendar (Some of which are mine, some other are my friend's)

Each Calendar is like a folder containing a list of events for one event

I want to list all event in a particular date cross calendar

Can I do this?

**Comment by makgr** [comment]
you may try this.
https://github.com/spatie/laravel-google-calendar/discussions/251#discussioncomment-2745273
